src/notification-banner/notification-banner.tsx

```diff
@@ -40,23 +46,48 @@ export const NBComponent = ({
     // =============================================================================
     // EVENT HANDLERS
     // =============================================================================
-    const handleDismiss = () => {
+    const handleDismiss = (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.stopPropagation();
         setVisible(false);
 
         if (dismissible && onDismiss) onDismiss();
     };
 
+    const handleBannerClick = (event: React.MouseEvent<HTMLInputElement>) => {
+        if (!onClick) return;
+        event.stopPropagation();
+        onClick();
+    };
+
     // =============================================================================
     // RENDER FUNCTIONS
     // =============================================================================
     if (!isVisible) return null;
 
+    const renderViewMore = () => (
+        <ViewMoreButton>
```

Comment:
semantically, this should be a `styled.button`

this allows keyboard users or screen readers to interact with the overall banner

likely need a `event.stopPropagation` to prevent triggering the main banner again